include/fiction/io/write_operational_domain.hpp

```diff
@@ -121,45 +124,94 @@ inline void write_operational_domain(const operational_domain<parameter_point, o
         throw std::invalid_argument("unsupported number of dimensions in the given operational domain");
     }
 
-    if (num_dimensions == 1)
+    if (!opdom.metric_values.empty())
     {
-        writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]), "operational status");
-    }
-    else if (num_dimensions == 2)
-    {
-        writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]),
-                          detail::sweep_parameter_to_string(opdom.dimensions[1]), "operational status");
-    }
-    else  // num_dimensions == 3
-    {
-        writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]),
-                          detail::sweep_parameter_to_string(opdom.dimensions[1]),
-                          detail::sweep_parameter_to_string(opdom.dimensions[2]), "operational status");
-    }
-
-    for (const auto& [sim_param, op_val] : opdom.operational_values)
-    {
-        // skip non-operational samples if the respective flag is set
-        if (params.writing_mode == write_operational_domain_params::sample_writing_mode::OPERATIONAL_ONLY &&
-            op_val == operational_status::NON_OPERATIONAL)
+        if (num_dimensions == 1)
+        {
+            writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]), "operational status", "ct");
+        }
+        else if (num_dimensions == 2)
         {
-            continue;
+            writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]),
+                              detail::sweep_parameter_to_string(opdom.dimensions[1]), "operational status", "ct");
         }
+        else  // num_dimensions == 3
+        {
+            writer.write_line(detail::sweep_parameter_to_string(opdom.dimensions[0]),
+                              detail::sweep_parameter_to_string(opdom.dimensions[1]),
+                              detail::sweep_parameter_to_string(opdom.dimensions[2]), "operational status", "ct");
+        }
```

Comment:
`"ct"` is hard-coded here. Would be cool to have a mapping from metrics to strings where this symbol is fetched. Also, I would spell it out as `"critical temperature"` since it will occur only once in the header of the file